src/index.html

```diff
@@ -1,18 +1,70 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section class="product-cards">
+      <article
+        class="product-card"
+        data-qa="card"
+      >
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="product-card__code-product">Product code: 195434</p>
+
+        <div class="product-card__reviews">
+          <div class="product-card__reviews-stars product-card__reviews-stars--4">
+            <div class="product-card__star"></div>
+            <div class="product-card__star"></div>
+            <div class="product-card__star"></div>
+            <div class="product-card__star"></div>
+            <div class="product-card__star"></div>
+          </div>
+
+          <p class="product-card__reviews-paragraph">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__price-label">Price:</p>
+
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+
+        <a
```

Comment:
The 'href' attribute value '#buy' doesn't seem to lead to any specific part of the page. If it's meant to be a placeholder, consider adding a comment to clarify this. If it's meant to link to a section where the user can purchase the product, ensure that the corresponding section exists and has an id of 'buy'.